posts/2022-08-24-open-liberty-with-cargo-tracker.adoc

```diff
@@ -0,0 +1,114 @@
+
+* <<maven_setup, Maven command line setup>> 
+* <<eclipse_setup, Eclipse IDE setup>>
+
+[#maven_setup]
+== Maven command line setup
+
+After cloning the project, you can quickly start Cargo Tracker through the command line with Maven. Navigate to the project base directory (where the pom.xml file is located) and run:
+
+[source]
+----
+mvn -P openliberty liberty:dev
+----
+ 
+The application should start without any thrown exceptions, and the startup messages such as the message-driven bean activation warnings are cosmetic and can be safely ignored. 
+
+[.img_border_dark]
+image::/img/blog/cargo-tracker-maven-start.png[Cargo Tracker Maven Output,width=70%,align="center"]
+
+[#eclipse_setup]
+== Eclipse IDE setup
+
+You'll need Eclipse IDE version 2022-03 or later to run the application (either the "Eclipse IDE for Java Developers" or "Eclipse IDE for Enterprise Java and Web Developers" packages will work). After cloning the project, import it into Eclipse by selecting the "File" menu and then "Import". Choose the "Existing Maven Projects" option inside the import menu. In the next step, select the repository base folder as the root directory and check the pom.xml under the "Projects" tab. Select "Finish" to complete the import process. 
+
+[.img_border_dark]
+image::/img/blog/cargo-tracker-eclipse-import.png[Cargo Tracker Eclipse Import,width=60%,align="center"]
+
+[.img_border_dark]
+image::/img/blog/cargo-tracker-eclipse-config.png[Cargo Tracker Eclipse Configuration,width=60%,align="center"]
+
+If you're working inside the Eclipse IDE, you'll need to install IBM's Liberty Tools feature. Following the steps in the https://github.com/OpenLiberty/liberty-tools-eclipse/blob/main/docs/installation/installation.md[installation guide] should allow you to access some convenient new resources and operations specific to Liberty. There's a https://openliberty.io/blog/2022/08/01/liberty-tools-eclipse.html[blog post] that covers the feature's capabilities in more detail, but for now we'll focus on the Liberty Dashboard. To run Cargo Tracker you'll need to right-click the `cargo-tracker` project and select the "Start..." option, which is second from the top. You can then run dev mode with the Open Liberty profile active by typing in `-Popenliberty`. 
+
+[.img_border_dark]
+image::/img/blog/cargo-tracker-eclipse-start.png[Cargo Tracker Eclipse Start,width=60%,align="center"]
+[.img_border_dark]
+image::/img/blog/cargo-tracker-eclipse-parameters.png[Cargo Tracker Eclipse Parameters,width=60%,align="center"]
+
+
+After starting with the custom parameters for the Open Liberty profile, the application should start inside the Eclipse terminal.
+
+=== Avoid trouble - set environment variables
+
+Since the Liberty Tools feature uses the well-known environment variables: **PATH** and **JAVA_HOME**, you will encounter problems if these variables are not set within your Eclipse IDE process, e.g. the `mvn` executable might not be found.
```

Comment:
```suggestion
Since the Liberty Tools feature uses the well-known environment variables `PATH` and `JAVA_HOME`, you will encounter problems if these variables are not set within your Eclipse IDE process, e.g. the `mvn` executable might not be found.
```

Reply:
use monospace for code entities as opposed to bold, which should be used for UI elements